bigtable-hbase-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/read/ScanAdapter.java

```diff
@@ -90,16 +90,17 @@ public RowFilter buildFilter(Scan scan, ReadHooks hooks) {
   @Override
   public Builder adapt(Scan scan, ReadHooks readHooks) {
     throwIfUnsupportedScan(scan);
-    // For gets, startRow == stopRow.  There's no need to create a new ByteString for stopRow
     RowFilter filter = buildFilter(scan, readHooks);
 
-    return ReadRowsRequest.newBuilder()
-        .setFilter(filter)
-        .setRows(RowSet.newBuilder()
-            .addRowRanges(
-              RowRange.newBuilder()
-                  .setStartKeyClosed(ByteString.copyFrom(scan.getStartRow()))
-                  .setEndKeyOpen(ByteString.copyFrom(scan.getStopRow()))));
+    RowSet.Builder rowSetBuilder = RowSet.newBuilder();
+    if (scan.isGetScan()) {
+      rowSetBuilder.addRowKeys(ByteString.copyFrom(scan.getStartRow()));
+    } else {
+      rowSetBuilder.addRowRanges(
+        RowRange.newBuilder().setStartKeyClosed(ByteString.copyFrom(scan.getStartRow()))
+            .setEndKeyOpen(ByteString.copyFrom(scan.getStopRow())));
```

Comment:
Indentation here seems a bit inconsistent. Personally I'm fine with the style the way it was (with leading periods) but in both cases the indentation seems off (4 spaces instead of 2)?